code/modules/mod/mod_ai.dm

```diff
@@ -127,9 +131,10 @@
 	if(!do_after(user, 5 SECONDS, target = src))
 		balloon_alert(user, "interrupted!")
 		return
+	var/mob/living/silicon/ai/ai = stored_ai.resolve()
```

Comment:
Check that `ai` is not null here, `resolve()` returns null if the datum is hard deleted.

You can probably just `qdel(src)` if it is.